apps/studio/src/pages/home/profile/Profile.tsx

```diff
@@ -275,6 +282,8 @@ const Profile: FunctionComponent = () => {
             { error: errors.firstName, element: firstNameRef.current },
             { error: errors.lastName, element: lastNameRef.current },
             { error: errors.companyName, element: companyNameRef.current },
+            { error: errors.isni, element: isniRef.current },
+            { error: errors.ipi, element: userIpiRef.current },
```

Comment:
Can remove `user` from ipi to follow suit with the refactor done in the other files.